test/integration/patients/worklist/schedule.js

```diff
@@ -903,6 +903,19 @@ context('schedule page', function() {
       .find('.js-bulk-edit')
       .should('contain', 'Edit 1 Action');
 
+    cy
+      .get('.schedule-list__table')
+      .find('.schedule-list__list-row .is-selected')
+      .should('have.length', 1)
+      .first()
+      .find('.js-select')
+      .click();
```

Comment:
Verifies that the `.is-selected` class is there when a list item initially loads as selected.